### PR TITLE
⭐️ new panic handler that reports client panics

### PR DESCRIPTION
**Pull request by chris-rock**
This adds a panic handler to allow authenticated clients to report panics up to the Mondoo platform.

### CODE REVIEW

File: providers-sdk/v1/upstream/health/errors.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+// sendPanic sends a panic to the mondoo platform for further analysis if the
+// service account is configured.
+// This function does not return an error as it is not critical to send the panic to the platform.
+func sendPanic(product, version, build string, r any, stacktrace []byte) {
+	// 1. read config
+	opts, err := config.Read()
+	if err != nil {
+		log.Info().Err(err).Msg("failed to read config")
```
Comment (preslavgerchev) [inline providers-sdk/v1/upstream/health/errors.go]: nit:
```suggestion
		log.Info().Err(err).Msg("panic handler> failed to read config")
```

gives us more context as to where that thing has failed when looking at the CLI

File: providers-sdk/v1/upstream/health/errors.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+		return
+	}
+
+	serviceAccount := opts.GetServiceCredential()
+	if serviceAccount == nil {
+		log.Info().Msg("no service account configured")
```
Comment (preslavgerchev) [inline providers-sdk/v1/upstream/health/errors.go]: nit:
```suggestion
		log.Info().Msg("panic handler> no service account configured")
```

File: providers-sdk/v1/upstream/health/errors.go [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+		Error: &ErrorInfo{
+			Message:    "panic: " + fmt.Sprintf("%v", r),
+			Stacktrace: string(stacktrace),
+		},
+	}
+
+	// 3. send error to mondoo platform
+	proxy, err := config.GetAPIProxy()
+	if err != nil {
+		log.Info().Err(err).Msg("failed to parse proxy setting")
```
Comment (preslavgerchev) [inline providers-sdk/v1/upstream/health/errors.go]: all of those feel like they should be error level, not info

Comment (chris-rock) [reply providers-sdk/v1/upstream/health/errors.go]: oh yes, forgot this to change back during testing

**Pull request merged**